dotnet/src/Connectors/Connectors.Memory.Weaviate/Http/ApiSchema/GraphGetRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Net.Http;
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Weaviate.Http.ApiSchema;
+
+internal sealed class GraphGetRequest
+{
+    public string Class { get; set; }
```

Comment:
Need to handle `CS8618` here, either initialize these properties or add a suppression (like in AzureCosmosDB for example).